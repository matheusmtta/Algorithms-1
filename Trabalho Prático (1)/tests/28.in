24 i
41 i
52 i
56 i
41 r
56 r
77 i
18 i
83 p
8 i
63 i
246 p
65 i
305 p
37 p
215 p
238 p
59 i
43 i
328 p
98 p
77 r
19 i
194 p
43 r
55 i
85 p
335 p
21 i
328 p
320 p
6 i
171 p
48 i
63 r
320 p
23 i
55 r
59 r
51 i
72 i
74 i
354 p
128 p
381 p
29 i
48 r
37 i
24 r
8 r
6 r
75 i
51 r
65 r
19 r
74 r
68 i
35 i
29 r
187 p
37 r
381 p
418 p
71 i
7 i
72 r
16 i
75 r
26 p
68 r
278 p
10 i
18 r
231 p
16 r
33 i
174 p
4 i
52 r
64 i
157 p
