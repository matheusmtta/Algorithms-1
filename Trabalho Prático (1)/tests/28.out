3
6
6
3
4
5
5
3
4
3
6
6
6
4
6
6
3
6
3
6
6
3
6
5
4
4
